index.html

```diff
@@ -145,7 +148,7 @@ <h3 class="card-heading">Languages</h3>
     <li id="frameworks">
       <div class="skills-card">
```

Comment:
- [ ] The mobile version of your cards sdo not match the figma design please work on adapting the technologies to align correctly please kindly work on fixing this :+1: 
- figma design 
![card-f](https://user-images.githubusercontent.com/89055502/236408219-d574c3ee-113d-4dfa-ad10-0744f65f34a3.png)
- your design
![card-s](https://user-images.githubusercontent.com/89055502/236408243-b1a33637-ddbf-4bf0-b0c4-3f9add49b74d.png)